tests/resources/main.tf

```diff
@@ -4,10 +4,70 @@
 
 module "landing_zone" {
   source                 = "terraform-ibm-modules/landing-zone/ibm//patterns//vpc//module"
-  version                = "6.0.0"
+  version                = "6.2.2"
   region                 = var.region
   prefix                 = var.prefix
   tags                   = var.resource_tags
   enable_transit_gateway = false
   add_atracker_route     = false
 }
+
+################################################################################
+# Resource Group
+################################################################################
+module "resource_group" {
+  source              = "terraform-ibm-modules/resource-group/ibm"
+  version             = "1.1.6"
+  resource_group_name = "${var.prefix}-rg"
+}
+
+#################################################################################
+# Secrets Manager resources
+#################################################################################
+
+locals {
+  sm_region            = module.secrets_manager.secrets_manager_region
+  secrets_manager_guid = module.secrets_manager.secrets_manager_guid
+}
+
+module "secrets_manager" {
```

Comment:
Don't create a new instance - the test should use our existing instance

Reply:
update